### PR TITLE
Reviewer EM: Add cause, effect and action to alarms

### PR DESCRIPTION
**Pull request by graemerobertson**

### CODE REVIEW

File: metaswitch/common/alarms_parser.py [hunk 1]
```diff
@@ -82,6 +85,9 @@ def parse_alarms_file(json_file):
      "Details length was greater than 255 characters in alarm {}".format(name)
                 assert len(level['description']) < 256, \
      "Description length was greater than 255 characters in alarm {}".format(name)
+                assert level['cause']
```
Comment (eleanor-merry) [inline metaswitch/common/alarms_parser.py]: Can you put some human readable text on the asserts?

Comment (eleanor-merry) [inline metaswitch/common/alarms_parser.py]: Actually, is assert the right thing here?

**Comment by eleanor-merry** [comment]
One comment, otherwise looks good

**Comment by chris-elford-metaswitch** [comment]
I've just had a chance to look at this. We currently have tests in alarms_parser.py (the one in the tests folder, not the real one) that check that we enforce certain constraints on our alarms. You've started enforcing some new constraints, so I think you need new tests.

**Comment by graemerobertson** [comment]
https://github.com/Metaswitch/python-common/pull/43

**Pull request merged**